external/googleapis/CMakeLists.txt

```diff
@@ -230,6 +230,8 @@ set(external_googleapis_installed_libraries_list
     google_cloud_cpp_iam_v1_iam_policy_protos
     google_cloud_cpp_iam_v1_options_protos
     google_cloud_cpp_iam_v1_policy_protos
+    google_cloud_cpp_iam_v2_deny_protos
```

Comment:
I think `google/cloud/iam` is already compiling these protos.  I believe this change introduces a second library that compiles the same protos.  That is basically an ODR violation waiting to happen.

If I am right, we need to refactor these common protos to a new library **or** we need to have `google/cloud/policytroubleshooter` depend on `google/cloud/iam`.  The first option seems more desirable, and it would be similar to the pure-protos library in `google/cloud/grafeas`.

Reply:
This change makes it so that we always build the v2 protos.

I think we can clean it up as part of #8022.